site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: tutorial
+title: Theming
+overview: In this tutorial, you will learn how to use the Dojo 2 theming and styling system to add a theme to the sample application.
+---
+
+{% section 'first' %}
+
+# Theming
+
+## Overview
+This tutorial will extend on previous tutorials where we created the basic biz-e-corp application. In this tutorial, we will adapt the app to allow it to be themed, write a theme and apply it to our application. This will be done using the powerful theming system that is baked into Dojo 2.
+
+## Prerequisites
+You can [download](../assets/007_themeing-initial.zip) the demo project and run `npm install` to get started.
+
+The `@dojo/cli` command line tool should be installed globally. Refer to the [Dojo 2 local installation](../000_local_installation/) article for more information.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+{% section %}
+
+## Making our widgets themeable
+
+{% task 'Make our widgets themeable' %}
+
+In order to theme our widgets, we must ensure that they each extent `ThemeableMixin` and should change their top level class name to `root`. `ThemeableMixin` provides us with a `this.classes` function that can lookup class names in the provided `theme` file and applies classnames appropaitely. We change the top level classnames to `root` in order to provice a predictable way to target the outernode of a iwgdte.
```

Comment:
extent -> extend

Reply:
Let's introduce the concept/convention of a `root` class before the instruction to use it?

Reply:
because it's not really a mandatory requirement, more a recommendation on best practice.